Service/GeneratorService.php

```diff
@@ -33,7 +33,7 @@ public function generate(GeneratorConfig $config): array
     private function getTypescriptUtilityFunctions(): array
     {
         return [
-            'const replaceRouteParams = (rawRoute: string, routeParams: Record<string, string|number>): string => {Object.entries(routeParams).forEach(([key, value]) => rawRoute = rawRoute.replace(`{${key}}`, value as string)); return rawRoute;}',
+            'const replaceRouteParams = (rawRoute: string, routeParams: Record<string, string|number|null>): string => {Object.entries(routeParams).forEach(([key, value]) => rawRoute = rawRoute.replace(`{${key}}`, value as string)); return rawRoute;}',
             'const appendQueryParams = (route: string, queryParams?: Record<string, string|number>): string => queryParams ? route + "?" + new URLSearchParams(queryParams as Record<string, string>).toString() : route;',
```

Comment:
Maybe we should also make `appendQueryParams` accept null value in case someone wants to manually pass null for a query parameter, @BolZer wdyt